holoviews/tests/ui/bokeh/test_callback.py

```diff
@@ -184,7 +192,7 @@ def test_multi_axis_tap(serve_hv):
 
     def test():
         assert s.xs == {'x': 11.560240963855422}
-        assert s.ys == {'y1': 18.642857142857146, 'y2': 78.71428571428572}
+        assert s.ys == {'y1': 18.642857142857146, 'y2': np.float64(78.71428571428572)}
```

Comment:
```suggestion
        assert len(s.ys) == 2
        assert np.isclose(s.ys["y1"], 18.642857142857146)
        assert np.isclose(s.ys["y2"], 78.71428571428572)
```

I assume this change is because of precision, and we don't need to be that precise here.